com.unity.render-pipelines.high-definition/Runtime/Lighting/ScreenSpaceLighting/SSGIDenoiser.cs

```diff
@@ -199,7 +207,8 @@ static void Denoise(CommandBuffer cmd, SSGIDenoiserParameters parameters, SSGIDe
             if (parameters.historyNeedsClear)
             {
                 // clear it to black if this is the first pass to avoid nans
-                CoreUtils.SetRenderTarget(cmd, resources.indirectDiffuseHistory, ClearFlag.Color, clearColor: Color.black);
+                CoreUtils.SetRenderTarget(cmd, resources.indirectDiffuseHistory0, ClearFlag.Color, clearColor: Color.black);
```

Comment:
For just clearing buffers it'd be better to use CoreUtils.ClearRenderTarget for the sake of clarity.